### PR TITLE
Cleanup of the DCM dts file so that it will boot

### PR DESCRIPTION
**Pull request by macorke**
A number of devices needed to be disabled to get the DCM to boot.

### CODE REVIEW

File: nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts [hunk 1]
```diff
@@ -26,13 +26,76 @@
...
+        status = "disabled";
+    };
 
     //*********************************************************************
     // Serial UART Setup
     //*********************************************************************
     // Xavier Module UART 3 -->  UART 1 (Debug Port)
     serial@0c280000 {
-        compatible = "nvidia,tegra184-uart", "nvidia,tegra114-hsuart", "nvidia,tegra20-uart";
+        compatible = "nvidia,tegra210-uart", "nvidia,tegra114-hsuart", "nvidia,tegra20-uart";
```
Comment (MikeLoder) [inline nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts:98]: Tegra210 is the Nano, you sure this shouldn't be tegra194?

File: nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts [hunk 2]
```diff
@@ -26,13 +26,76 @@
...
+        vi@15c10000 {
+            status = "disabled";
+        };
+        dpaux@155F0000 {
+            status = "disabled";
+        };
+    };
+
+    // Disable the ethernet controller
+    ether_qos@2490000 {
```
Comment (MikeLoder) [inline nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts:49]: Won't this take down the ethernet to the CTM?

File: nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts [hunk 3]
```diff
@@ -26,13 +26,76 @@
...
+        status = "disabled";
+    };
+
+    // Disable the framebuffer memory reserve
+    reserved-memory {
```
Comment (MikeLoder) [inline nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts:54]: I expect you will need the framebuffer for the LCD

File: nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts [hunk 4]
```diff
@@ -139,12 +200,11 @@
             nvidia,rx-clk-tap-delay = <6>;
         };
     };
-
     //*********************************************************************
     // GPIO Input Buttons, configured to trigger keyboard events.
     //*********************************************************************
 
-    gpio-keys {
+   gpio-keys {
```
Comment (m3harvey) [inline nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts:207]: Doesn't this make the tabbing wrong?

File: nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts (hunk 2 above)
Comment (macorke) [reply nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts:49]: This was a source of one of the kernel crashes, needs to be investigated.

File: nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts (hunk 3 above)
Comment (macorke) [reply nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts:54]: We are getting these at startup , this eliminates the error. Needs to be investigated.
] OF: fdt:Reserved memory: failed to reserve memory for node 'fb1_carveout': base 0x0000000B
[    0.000000] OF: fdt:Reserved memory: failed to reserve memory for node 'fb1_carveout': base 0x0000000B
[    0.000000] OF: fdt:Reserved memory: failed to reserve memory for node 'fb2_carveout': base 0x0000000B
[    0.000000] OF: fdt:Reserved memory: failed to reserve memory for node 'fb2_carveout': base 0x0000000B
[    0.000000] OF: fdt:Reserved memory: failed to reserve memory for node 'fb3_carveout': base 0x0000000B
[    0.000000] OF: fdt:Reserved memory: failed to reserve memory for node 'fb3_carveout': base 0x0000000B

File: nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts (hunk 1 above)
Comment (macorke) [reply nvidia/platform/t19x/galen/kernel-dts/tegra194-p2888-0001-royaloak-dcm.dts:98]: There didn't seem to be a tegra194-uart. Something is still wrong with this setup.

**Pull request merged**